VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionRequestFields.java

```diff
@@ -17,9 +22,26 @@
     /**
      * Get the value of the query parameter.
      *
-     * @return The query parameter string
+     * @return The query parameter string.
      */
     public String getQueryParamValue() {
         return queryParamValue;
     }
+
+    /**
+     * Generate the list of query param values to send for the requested_fields param.
+     *
+     * @param config The Config object to use for conditional param value additions.
+     * @return List of requested fields for query param.
+     */
+    public static List<String> getRequestedFieldsList(@NonNull Config config) {
+        final List<String> requestedFields;
+        if (config.isDiscussionProfilePicturesEnabled()) {
+            requestedFields = Collections.singletonList(
+                    DiscussionRequestFields.PROFILE_IMAGE.getQueryParamValue());
```

Comment:
Since this method is now defined _inside_ the `DiscussionRequestFields` class, you don't need to declare it here in order to resolve it's constants.